src/OrchardCore.Modules/OrchardCore.Forms/Views/Form.Wrapper.cshtml

```diff
@@ -20,14 +20,17 @@
 
     @await DisplayAsync(Model.Metadata.ChildContent)
 </form>
+
 <script at="Foot">
-    var formId = '#'+ '@formId'
-    $(formId).on('click', function () {
-        console.log(window.validatorObject)
-        $("<input />").attr("type", "hidden")
-            .attr("name", "validateRules")  
-            .attr("value", JSON.stringify(window.validatorObject))
-            .appendTo(formId);
-        return true;
+    $(function () {
```

Comment:
It's best to use the `jQuery` variable directly instead of the dollar sign because the latter can be overloaded. You can wrap this whole block into a 
```js
(function ($) {
...
})(jQuery);
```
to work around this. See: https://orcharddojo.net/orchard-resources/CoreLibrary/DevelopmentGuidelines/BestPractices/JavaScript